src/ui/interface.js

```diff
@@ -1679,4 +1701,4 @@ var UI = class UI {
 			}
 		});
 	}
-};
+};
```

Comment:
All the files should also already end with a newline, let's keep that consistent.